packages/@react-spectrum/menu/src/MenuTrigger.tsx

```diff
@@ -67,14 +67,43 @@ export function MenuTrigger(props: SpectrumMenuTriggerProps) {
     isOpen
   });
 
+  let isMobile = useMediaQuery('(max-width: 700px)');
   let menuContext = {
     ...menuProps,
     focusStrategy,
     onClose,
     closeOnSelect,
     autoFocus: true,
-    wrapAround: true
+    wrapAround: true,
+    UNSAFE_style: {
```

Comment:
Tray doesn't take care of this already?

Reply:
not at the moment. do you think it's safe to do that? wasn't sure if trays could have more than one child or something